test/e2e/specs/fakerp/admin.go

```diff
@@ -103,4 +103,23 @@ var _ = Describe("Openshift on Azure admin e2e tests [AzureClusterReader][Fake]"
 		By("comparing the update blob before and after an update")
 		Expect(reflect.DeepEqual(before, after)).To(Equal(true))
 	})
+
+	It("should be possible for an SRE to reimage and restart a VM in a scale set", func() {
+		if os.Getenv("RESOURCEGROUP") == "" {
+			Expect(errors.New("RESOURCEGROUP is not set")).ToNot(HaveOccurred())
+		}
+		cli, err := azure.NewClientFromEnvironment(false)
+		Expect(err).ToNot(HaveOccurred())
+
+		By("Reading the cluster state")
+		before, err := cli.OpenShiftManagedClustersAdmin.Get(context.Background(), os.Getenv("RESOURCEGROUP"), os.Getenv("RESOURCEGROUP"))
+		Expect(err).NotTo(HaveOccurred())
+		Expect(before).NotTo(BeNil())
+
+		By("Executing reimage and restart on a vm in the cluster")
+		update, err := cli.OpenShiftManagedClustersAdmin.ReimageAndRestartAndWait(context.Background(), os.Getenv("RESOURCEGROUP"), os.Getenv("RESOURCEGROUP"), "ss-master", "0")
+		Expect(err).NotTo(HaveOccurred())
+		Expect(update.StatusCode).To(Equal(http.StatusOK))
+		Expect(update).NotTo(BeNil())
+	})
```

Comment:
You need to prove that the instance was reimaged.
You should then run the cluster sanity checks.

Reply:
done